x-pack/test/alerting_api_integration/security_and_spaces/scenarios.ts

```diff
@@ -154,13 +154,50 @@ const Space1AllWithRestrictedFixture: User = {
   },
 };
 
+/**
+ * This user is needed to test system actions.
+ * In x-pack/test/alerting_api_integration/common/plugins/alerts/server/action_types.ts
+ * we registered a system action type which requires access to Cases. This user has
+ * access to Cases only in the Stack Management. The tests use this user to
+ * execute the system action and verify that the authorization is performed
+ * as expected
+ */
+const CasesAll: User = {
+  username: 'cases_all',
+  fullName: 'cases_all',
+  password: 'cases_all',
+  role: {
+    name: 'cases_all_role',
+    elasticsearch: {
+      indices: [
+        {
+          names: [`${ES_TEST_INDEX_NAME}*`],
+          privileges: ['all'],
+        },
+      ],
+    },
+    kibana: [
+      {
+        feature: {
+          generalCases: ['all'],
```

Comment:
If you want me to register a feature in `x-pack/test/alerting_api_integration/common/plugins/actions_simulators/server/plugin.ts` and use it instead of using Cases let me know.